src/renderer/viz/expressions/RampImage.js

```diff
@@ -43,9 +47,8 @@ export default class RampImage extends Base {
     }
 
     getLegendData (options) {
-        const config = Object.assign({}, DEFAULT_OPTIONS, options);
         const type = this.input.type;
-        const legendData = this.input.getLegendData(config);
+        const legendData = this.input.getLegendData(options);
```

Comment:
Why have we omitted here the previous DEFAULT_OPTIONS?

Reply:
By default, the default options (`othersLabel` and `samples`) are the same, but they're defined in the constructor of the nested expression. I forgot to added in the PR description (now it's added), but this comes from this issue: https://github.com/CartoDB/carto-vl/issues/1291.